docs/arf.md

```diff
@@ -2456,6 +2517,8 @@ practice this will often mean that the identity document is a photo ID,
 and the presentation must consequently be done in proximity and be
 supervised, or done remotely and supported by PAD.
 
+[**The PID should contain a photo by default. The fact it doesn't is an ommission and needs to be corrected.**]
+
```

Comment:
According to the recently published Implementing Act on PID and EAA, inclusion of a User portrait is optional.